src/python_testing/test_testing/test_TC_TMP_2_1.py

```diff
@@ -160,7 +160,7 @@ def test_spec_to_attribute_cache(test_spec: TestSpec) -> Attribute.AsyncReadTran
 
 
 def main():
-    test_runner = MockTestRunner('TC_TMP_2_1', 'TC_TMP_2_1', 'test_TC_TMP_2_1')
+    test_runner = MockTestRunner('TC_TMP_2_1', 'TC_TMP_2_1', 'test_TC_TMP_2_1', 1)
```

Comment:
nit, was this accidentally picked up?

Reply:
Nope - I added a new parameter to the init function to specify the endpoint explicitly. So it had to be added here (only other test that uses this currently)